### ISSUE TITLE
jq for shorter syntax

### ISSUE THREAD

**Issue by estahn** [garethr/kubetest#28]
It would be great if we could use jq for quick evaluations, e.g.

```
assert_equal(jq(spec, "spec.strategy.rollingUpdate.maxUnavailable"), 0)
```

**Comment by jsleeio** [comment]
I like it. My experience with `kubetest` has included a lot of time spent trying to avoid tests failing due to Skylark errors when fields are not present in the input object, and I think your idea would help simplify those tests

**Comment by estahn** [comment]
@jsleeio I agree, we have the same experience. I wrote @garethr an email if this project is still actively maintained. Unfortunately, I haven't heard back from him.

I can add that feature but not sure if that would ever get merged.
